extensions/big-data-cluster/.gitignore

```diff
@@ -1 +0,0 @@
-*.vsix
```

Comment:
Why are you deleting all this other stuff? Bad merge?

Reply:
The all other stuff is no longer needed, and I was requested to remove by Alan Ren

Reply:
@alanrenmsft Comments? Why would we be removing .gitignore/.vscodeignore/README.md? And the other stuff is ok to completely remove?

Reply:
@alanrenmsft Any thoughts?

Reply:
you need to keep this file

Reply:
fixed.